tools/test/audio/std_utils/stdnotch_get.m

```diff
@@ -43,7 +43,7 @@
 
 target_q = 2.1;
 bw = (2*fn/fs)/target_q;
-if exist('iirnotch.m') == 2
+if exist(fullfile('iirnotch.m'),'file') == 2
```

Comment:
Unfortunately this is not sufficient, it should be e.g. fullfile('.', ' iirnotch.m'). When I tried, just s = fullfile('iirnotch.m') there's no path added to beginning.